playground/TestCommon/TestArrayMaze.cs

```diff
@@ -0,0 +1,84 @@
+using Common;
+
+namespace TestCommon;
+
+public class TestArrayMaze
+{
+    [Test]
+    [TestCase(5, 5, 5)]
+    [TestCase(1, 2, 3)]
+    [TestCase(3, 2, 1)]
+    [TestCase(1, 10, 50)]
+    [TestCase(12, 23, 44)]
+    public void TestConstructor(int sizeX, int sizeY, int blockSize)
+    {
+        //Arrange    
+        var random = new Random(sizeX + sizeY + blockSize);
+        var storage = new[] { random.NextInt64() };
+
+        //Act
+        var maze = new ArrayMaze(sizeX, sizeY, blockSize, storage);
+
+        //Assert
+        Assert.Multiple(() =>
+        {
+            //Assert
+            Assert.That(maze.MazeSizeX, Is.EqualTo(sizeX), nameof(maze.MazeSizeX));
+            Assert.That(maze.MazeSizeY, Is.EqualTo(sizeY), nameof(maze.MazeSizeY));
+            Assert.That(maze.BlockSize, Is.EqualTo(blockSize), nameof(maze.BlockSize));
+            Assert.That(maze.Storage, Is.EqualTo(storage), nameof(maze.Storage));
+        });
+    }
+
+    [Test]
```

Comment:
Please remove this TestAttribute,
NUnit engine understands this method is a test because it has TestCaseAttribute(s).
Use either [Test] or [TestCase], not both